mm2src/coins/hd_wallet_storage/mod.rs

```diff
@@ -321,80 +303,61 @@ mod tests {
-        let rick_user1_device0_db =
-            HDWalletCoinStorage::init_with_rmd160(&ctx, "RICK".to_owned(), user1_rmd160, device0_rmd160)
-                .await
-                .expect("!HDWalletCoinStorage::new");
-        let morty_user0_device0_db =
-            HDWalletCoinStorage::init_with_rmd160(&ctx, "MORTY".to_owned(), user0_rmd160, device0_rmd160)
-                .await
-                .expect("!HDWalletCoinStorage::new");
-
-        rick_user0_device0_db
-            .upload_new_account(rick_user0_device0_account0.clone())
+        let rick_device0_db = HDWalletCoinStorage::init_with_rmd160(&ctx, "RICK".to_owned(), device0_rmd160)
             .await
-            .expect("!HDWalletCoinStorage::upload_new_account: RICK user=0 device=0 account=0");
-        rick_user0_device0_db
-            .upload_new_account(rick_user0_device0_account1.clone())
+            .expect("!HDWalletCoinStorage::new");
+        let rick_device1_db = HDWalletCoinStorage::init_with_rmd160(&ctx, "RICK".to_owned(), device1_rmd160)
             .await
-            .expect("!HDWalletCoinStorage::upload_new_account: RICK user=0 device=0 account=1");
-        rick_user0_device1_db
-            .upload_new_account(rick_user0_device1_account0.clone())
+            .expect("!HDWalletCoinStorage::new");
+        let morty_device0_db = HDWalletCoinStorage::init_with_rmd160(&ctx, "MORTY".to_owned(), device0_rmd160)
             .await
-            .expect("!HDWalletCoinStorage::upload_new_account: RICK user=0 device=1 account=0");
-        rick_user1_device0_db
-            .upload_new_account(rick_user1_device0_account0.clone())
+            .expect("!HDWalletCoinStorage::new");
+
+        rick_device0_db
+            .upload_new_account(rick_device0_account0.clone())
+            .await
+            .expect("!HDWalletCoinStorage::upload_new_account: RICK device=0 account=0");
+        rick_device0_db
+            .upload_new_account(rick_device0_account1.clone())
+            .await
+            .expect("!HDWalletCoinStorage::upload_new_account: RICK device=0 account=1");
+        rick_device1_db
+            .upload_new_account(rick_device1_account0.clone())
             .await
-            .expect("!HDWalletCoinStorage::upload_new_account: RICK user=1 device=0 account=0");
-        morty_user0_device0_db
-            .upload_new_account(morty_user0_device0_account0.clone())
+            .expect("!HDWalletCoinStorage::upload_new_account: RICK device=1 account=0");
+        morty_device0_db
+            .upload_new_account(morty_device0_account0.clone())
             .await
-            .expect("!HDWalletCoinStorage::upload_new_account: MORTY user=0 device=0 account=0");
+            .expect("!HDWalletCoinStorage::upload_new_account: MORTY device=0 account=0");
 
         // All accounts must be in the only one database.
         // Rows in the database must differ by only `coin`, `mm2_rmd160`, `hd_wallet_rmd160` and `account_id` values.
```

Comment:
Shouldn't `mm2_rmd160` be removed from this comment, there is another comment with it too.